server.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const app = express();
+
+app.get("/ping", (req, res) => {
+  res.send("Hello world!");
+});
+
+app.listen(3000);
```

Comment:
It would be better to make the port number configurable through environment variables. This will make your application more flexible in different environments. You can do this by changing the listen line to: `app.listen(process.env.PORT || 3000);` [medium]